src/condition/size_condition.py

```diff
@@ -0,0 +1,11 @@
+import pandas as pd
+
+from src.condition.i_condition import ICondition
+
+class SizeCondition(ICondition):
+    def __init__(self, num_chars: int):
+        self.num_chars = num_chars
+        
+    def execute(self, row: pd.DataFrame) -> bool:
+        lengths = row.apply(len)
```

Comment:
What does this do?

Reply:
This applies the length function to each entry in the row and creates a row of results